### PR TITLE
Development

### PR DESCRIPTION
**Pull request by pereslop**

### CODE REVIEW

File: app/assets/stylesheets/posts.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
   }
   .post-photo {
     width: 700px;
-    .post-actions {
+    .post_actions {
```
Comment (Snick555) [inline app/assets/stylesheets/posts.scss]: please use everywhere the same syntax, underscore or gyphen

File: app/controllers/account/posts_controller.rb [hunk 2]
```diff
@@ -13,6 +13,7 @@ def create
 
   def show
     @post = resource
+    @posts = collection
     respond_to do |format|
```
Comment (Snick555) [inline app/controllers/account/posts_controller.rb:17]: I think you can write here just `render ...` instead `respond_to` block

File: app/controllers/account/users/posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  private
+
+
+  def post_update_params
+    params.require(:post).permit(:content)
+  end
+
+  def collection
+    @user = User.find(params[:id])
+    @posts = @user.posts.ordered
```
Comment (Snick555) [inline app/controllers/account/users/posts_controller.rb]: why do you set to a variable here?

File: app/models/application_record.rb [hunk 4]
```diff
@@ -1,12 +1,19 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
+  def self.next_for(obj)
```
Comment (Snick555) [inline app/models/application_record.rb:4]: do you really need these two methods?

File: app/views/account/users/posts/update_gallery.js.erb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+$('#gallery-modal').modal('show');
```
Comment (Snick555) [inline app/views/account/users/posts/update_gallery.js.erb:1]: what do you think about CoffeeScript in js views? :)

File: app/views/account/users/show.html.slim [hunk 6]
```diff
@@ -14,9 +14,10 @@
...
         - if @user.posts
           #posts
-            = render @user.posts.ordered
+            = render partial: 'account/posts/post', collection: @user.posts, as: :post, locals: { user_post: true }
+            =paginate @posts, params: {controller: 'users', action: 'show'}
         -else
           h3 There no posts, your can be first!!!
-      =paginate @posts, params: {controller: 'users', action: 'show'}
-= render 'account/posts/modal', post: @posts.first
+- if @posts
```
Comment (Snick555) [inline app/views/account/users/show.html.slim]: please write it in one line

**Review by Snick555 — CHANGES_REQUESTED** [review]
Please add to ignore all coverage files

File: app/views/account/users/posts/update_gallery.js.erb (hunk 5 above)
Comment (pereslop) [reply app/views/account/users/posts/update_gallery.js.erb:1]: i think it is not 'coffee', it's just bottstrap-jquery methods, or I don't understand this

File: app/controllers/account/users/posts_controller.rb (hunk 3 above)
Comment (pereslop) [reply app/controllers/account/users/posts_controller.rb]: because i need a collection for displaying 'next-prev' posts now

**Pull request merged**